nipoppy/workflows/pipeline.py

```diff
@@ -488,12 +488,15 @@ def submit_hpc_job(self, participants_sessions):
         except NotImplementedError as e:
             self.logger.info(
                 f"pysqa has not implemented returning the array job ID for SGE yet! "
```

Comment:
```suggestion
                "pysqa has not implemented returning the array job ID for SGE yet! "
```
Not an f-string for this part (pre-commit complaining)